src/QuadVelocityController.cpp

```diff
@@ -129,77 +128,38 @@ bool QuadVelocityController::update(const ros::Time& time,
 
 // Get a valid transform, blocks until the transform is received or
 // the request takes too long and times out
-bool QuadVelocityController::getTransformAfterTime(
-        const ros::Time& time,
-        geometry_msgs::TransformStamped& transform,
-        const ros::Time& latest_time_allowed) {
+bool QuadVelocityController::waitForTransform(geometry_msgs::TransformStamped& transform) {
 
     // Catch TransformException exceptions
     try
     {
+        ros::Time time_at_start = ros::Time::now();
```

Comment:
Agreed. There is no problem with doing this because we get the latest transform anyway. So if two transforms come in after the last processed transform the latest will still be used and we won't miss transforms if running too slow.